src/less/core/imagery.less

```diff
@@ -0,0 +1,33 @@
+.doc-container {
+    .imagery {
+        h2,
```

Comment:
h2 and h3 should have `font-weight: 700` by default, as they are Swedbank Headline font?